pallets/common/src/traits/portfolio.rs

```diff
@@ -52,4 +52,8 @@ pub trait PortfolioSubTrait<Balance> {
     /// * `ticker` - Ticker of the token to unlock
     /// * `amount` - Amount of tokens to unlock
     fn unlock_tokens(portfolio: &PortfolioId, ticker: &Ticker, amount: &Balance) -> DispatchResult;
+
+    /// Transfer some funds to given portfolio.
+    #[cfg(feature = "runtime-benchmarks")]
+    fn fund_portfolio(portfolio: &PortfolioId, ticker: &Ticker, amount: Balance) -> DispatchResult;
```

Comment:
This could be a simple helper function in `benchmarking.rs`. Other benchmarks, if required, could import that. No trait method required.

Reply:
I need to access this from the settlement pallet, that's why It is added there as I don't have direct access to the portfolio pallet from the settlement besides the trait access. How do you suggest to achieve this?

Reply:
Can you access `pallet_portfolio::PortfolioAssetBalances::<T>`?

Reply:
You have a point. There is no direct way to portfolio methods that have the portfolio trait parameter. So let's have it wired through the identity module.